src/Uno.UI/Controls/NSStringAttributesHelper.macOS.cs

```diff
@@ -43,7 +43,7 @@ static NSStringAttributesHelper()
 			TextDecorations textDecorations
 		)
 		{
-			return _getAttributes((fontWeight, fontStyle, fontFamily, foreground, fontSize, characterSpacing, baseLineAlignment, textDecorations, 12.0f /* macOS TODO */));
+			return _getAttributes((fontWeight, fontStyle, fontFamily, foreground, fontSize, characterSpacing, baseLineAlignment, textDecorations));
```

Comment:
Aha - I'm guessing this was the source of the bug?

Reply:
Not directly.  It was changed because it's not a correct value, but the bug is "deeper" than that.